### PR TITLE
Update TransactionsList.js

### LINKED ISSUE #2: Add currency sign like $ at starting of amount
i have solved this issue assign it to me with hacktoberfest tag

### PR DESCRIPTION
**Pull request by khadimhussain99**
Fixes #2 
added currency sign in amount
sample-1

### CODE REVIEW

**Comment by MohitBansal321** [comment]
Great Work But make sure to write a relevant commit message
e.g Add dollar sign before amount in [file/filename]

File: client/src/components/TransactionsList.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default function TransactionsList({ data, fetchTransctions, setEditTransa
               month.transactions.map((row) => (
                 <TableRow key={row._id} sx={{ "&:last-child td, &:last-child th": { border: 0 } }}>
                   <TableCell align="center" component="th" scope="row">
-                    {row.amount}
+                    {`$${row.amount}}
```
Comment (MohitBansal321) [inline client/src/components/TransactionsList.js:82]: You did not complete the template literal I merged this PR 
I will do it myself
```
 {`$${row.amount}`}
```

Comment (khadimhussain99) [reply client/src/components/TransactionsList.js:82]: oh sorry buddy this is my first pr that's why i am little bit excited and forget to complete template literals.

**Comment by khadimhussain99** [comment]
Thanks.

File: client/src/components/TransactionsList.js (hunk 1 above)
Comment (MohitBansal321) [reply client/src/components/TransactionsList.js:82]: No issue

**Pull request merged**